blockly-ws-search/workspace_search.js

```diff
@@ -387,7 +424,7 @@ class WorkspaceSearch {
       return;
     }
     const searchGroup = this.getSearchPool_();
-    const isBlockMatch = this.isBlockMatch_;
+    const isBlockMatch = this.isBlockMatch_.bind(this);
```

Comment:
This bind is not necessary, isBlockMatch does not use `this`.
However, we could employ arrow function to avoid needing to make a variable for isBlockMatch, but I'll address in in another PR (because I should also refactor a bit in isBlockMatch_).

Reply:
Backtrack slightly, I forgot I add in `this.caseSensitive` to function. I'm making a PR that would fix. If trying to fix it here, I would recommend instead using arrow function, or leaving as I'll be fixing it.

Reply:
Yeah I was trying to fix bug I found in that. And ok I can revert and leave for you to fix in a different PR